docker-compose.yml

```diff
@@ -21,6 +21,9 @@
 ###
 mongo:
   image: mongo:latest
+  ports:
```

Comment:
Why are you exposing these ports to local host? This is not needed.

Docker container `mainflux-server` should be capable to directly access container `mongo`, no need need for Mongo DB ports exposed to the host.

Problem is of course that your Mongo client on your dev host will not work - you have to log into container and check DB contents.

But anyway - for dev I would expect that everybody use local MongoDB on host, and Docker is for deployments...